src/components/znkQuestionReport/controllers/znkReport.controller.js

```diff
@@ -0,0 +1,71 @@
+(function (angular) {
+    'use strict';
+
+    angular.module('znk.infra.znkQuestionReport').controller('znkReportCtrl',
+        function($log, $mdDialog, $timeout, $http, ENV, AuthService, MailSenderService, reportData) {
+            'ngInject';
+
+            var self = this;
+            var userAuth = AuthService.getAuth();
+            var MAIL_TO_SEND = 'ofir@zinkerz.com';
+            var TEMPLATE_KEY = 'reportQuestion';
+            var EMAIL_SUBJECT = 'Report Question';
+
+            self.success = false;
+            self.reportData = reportData;
+            self.reportData.app = ENV.firebaseAppScopeName.split('_')[0].toUpperCase();
+            self.reportData.email = userAuth.auth.email;
+            self.reportData.message = 'Hello Support,\r\n' +
+                                        'I\'ve noticed the following error in this question:\r\n';
+
+            this.sendFrom = function () {
+                if (self.reportForm.$valid) {
+                    self.startLoader = true;
+                    self.reportData.email = self.reportData.email ? self.reportData.email : userAuth.auth.email;
```

Comment:
there might be cases that we will not have email (facebook users who disabled email access). Make sure to support that